src/components/Practice/PhrasesInAction.jsx

```
@@ -6081,17 +6082,28 @@ const PhrasesInAction = ({
           )}
           <div
             style={{
-              width: "90%",
-              height: "90%",
+
+              width: isMobile ? "95%" : "90%",
```

ESLint

[error] Delete `⏎`

(prettier/prettier)